adobe/spectrum-web-components
# fix(number-field): show decimal on iPad minimized keyboard

## Description

This PR changes the inputMode of the `NumberField`'s `input` to `decimal` on iPad so the minimized keyboard can show the decimal point. 

## Related issue(s)

- https://github.com/adobe/spectrum-web-components/issues/4783

## Motivation and context

Currently an iPad user can't use the minimized keyboard to type in decimal values (i.e. 2.5).

## How has this been tested?




    1. Go to the decimals numberfield storybook ([here](https://imizga-i-pad-os-decimal-missing--spectrum-web-components.netlify.app/storybook/?path=/story/number-field--decimals))
    2. Click on the input itself to see the iPad's keyboard
    3. Pinch the keyboard to minimize it
    4. Observe that the keyboard now shows a decimal point
  
    1. Go to an editable slider's storybook ([here](https://imizga-i-pad-os-decimal-missing--spectrum-web-components.netlify.app/storybook/?path=/story/number-field--decimals))
    2. Click on the input itself to see the iPad's keyboard
    3. Pinch the keyboard to minimize it
    4. Observe that the keyboard now shows a decimal point


## Screenshots (if appropriate)
![simulator_screenshot_A78C42F2-2EBC-452B-8564-CE398B04C020](https://github.com/user-attachments/assets/71636ec1-4605-44b7-8980-ae2096485df8)


![simulator_screenshot_6D7D8B36-1359-4B82-B172-619A0627EF13](https://github.com/user-attachments/assets/ed12caed-fa7e-41fc-9e6c-bbec8532080a)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
